recipes/phylommand/build.sh

```diff
@@ -0,0 +1,31 @@
+cd src
+
+# Link flags are on the wrong line, leads to problems with NLOPT enabled.
+sed -i.bak 's/argv_parser.o \$(TREEATORLINKFLAGS)$/argv_parser.o/' Makefile 
+sed -i.bak 's/\$(PP) -o treeator \$(OTREEATOR)/\$(PP) -o treeator \$(OTREEATOR) \$(TREEATORLINKFLAGS)/' Makefile
+
+# CFLAGS, CPPFLAGS, and LDFLAGS are missing
+sed -i.bak '/\$(PP).*-c/ s/-c/$(CFLAGS) -c/' Makefile
+sed -i.bak '/\$(CC).*-c/ s/-c/$(CFLAGS) -c/' Makefile
+sed -i.bak '/\$(PP).*-o/ s/-o/$(LDFLAGS) -o/' Makefile
+
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+
+if [[ "$(uname)" == "Linux" ]] ; then
+    PP="$GXX -std=c++11"
```

Comment:
Use `$CXXFLAGS` - that includes the correct std settings (conda builds with c++17)

Reply:
Also, doesn't need a switch. Just use `PP="$CXX $CXXFLAGS"`

Reply:
Great, thanks for the help!